ONSdigital/eq-author-app
# Change to the qCode context to destructure additionalAnswer qCode

### Context of PR ###

qCode Table wasn't throwing an error when there was duplicate qCodes for otherLabels when using Radio or Checkbox. Adjusted `QCodeContext` so it now destructures out `additionalAnswer` and then grabs and renames the qCode. It then remaps through to check for `additionalAnswer` qCodes.

### How to review ###

- Add some Radio and Checkbox answers with `other` options.
- Add qCodes in the qCodes table.
- Create duplicate qCodes and check the error gets thrown.

What to do after everything is green
    *   Bring this branch up-to-date with Origin/Master
    *   If there are a lot of commits or some are not helpful to read, squash them down
    *   Click the Merge button on this pull request
    *   Does this change mean the Capability examples questionnaire should be updated?
    *   Move the Jira ticket for this task into the next stage of the process
